cloud-importer-openshift-local-aws-task.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
This could be `tekton.dev/v1`, v1 has been around for long time so IMHO it's safe to use. Migration guide is here https://tekton.dev/docs/pipelines/migrating-v1beta1-to-v1/ but I don't see anything incompatible.